package.json

```diff
@@ -63,26 +64,20 @@
     "body-parser": "^1.18.2",
     "bootstrap": "^4.1.1",
     "bsert": "0.0.4",
-    "bsock": "bcoin-org/bsock",
+    "bsock": "0.1.2",
     "bsock-middleware": "^1.1.3",
     "bweb": "0.0.1",
     "compression-webpack-plugin": "^1.1.11",
     "cors": "^2.8.4",
     "css-loader": "^0.28.7",
     "dotenv": "^5.0.1",
     "effects-middleware": "^1.0.1",
-    "eslint": "^4.9.0",
-    "eslint-config-prettier": "^2.6.0",
-    "eslint-import-resolver-webpack": "^0.8.3",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.4.0",
     "express": "^4.16.2",
     "extract-text-webpack-plugin": "^3.0.1",
-    "fetch-mock": "^5.13.1",
     "file-loader": "^1.1.5",
     "font-awesome": "^4.7.0",
+    "hs-client": "0.0.2",
+    "hsd": "handshake-org/hsd#bundle",
```

Comment:
I don't think that's necessary to be honest. It doesn't really affect the application and it could confuse things for someone to read about a temporary dependency. We had a lot of this before we had bcoin commit access too and it's even something that's done in some other bcoin-org projects.